src/me/mrCookieSlime/Slimefun/hooks/github/IntegerFormat.java

```diff
@@ -19,7 +19,7 @@ public static String formatBigNumber(int i) {
 	
 	public static Date parseGitHubDate(String str) {
 		try {
-			return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(str.replace("T", " ").replace("Z", ""));
+			return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(str.replace('T', ' ').replace("Z", ""));
```

Comment:
You missed the Z :)

Reply:
No I didn't, a char is by definition a single character. You can't do an empty char per say so it's best to just do an empty string here. And you can't do the "Z" alone as it would then not be matching CharSequences and fail compilation.

Reply:
Right, didn't see the right-side was an empty string.